server/lib/ReportManager.js

```diff
@@ -52,7 +52,9 @@ const defaults = {
 };
 
 // Constants
-const SAVE_DIR = path.resolve(path.join(__dirname, '../reports/'));
+const SAVE_DIR = process.env.REPORT_DIR;
```

Comment:
I just realized that defining this in `.env.development` will make it hard for developers to work on both linux and windows.  

I propose that we default to a temporary directory of the save directory isn't defined.  We can do it like this: 
```js
const SAVE)_DIR = process.env.REPORT_DIR || tempy.directory();
```
We have the [tempy module](https://github.com/IMA-WorldHealth/bhima/blob/master/package.json#L99) already for our tests. The documentation is [here](https://www.npmjs.com/package/tempy#tempydirectory).

We could also add a "debug()" statement that warns the developers that the `SAVE_DIR` is not defined, so it will use a temporary directory.

You would be able to remove the line from protractor's configuration as well.